english/dig/dig.md

```diff
@@ -2,7 +2,7 @@ You find that the walls are very thin, so you quickly break through.
 
 The effort has left you exhausted, but outside, you see a forest.
 
-A big bird approaches.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.